Content.Server/SimpleStation14/Jukebox/JukeboxSystem.cs

```diff
@@ -0,0 +1,352 @@
+using Content.Shared.SimpleStation14.Jukebox;
+using Content.Shared.SimpleStation14.Prototypes;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+using Robust.Server.GameObjects;
+using Content.Server.DeviceLinking.Systems;
+using Robust.Shared.Random;
+using Content.Shared.Power;
+using System.Linq;
+using Content.Server.Power.EntitySystems;
+
+namespace Content.Server.SimpleStation14.Jukebox;
+
+public sealed partial class JukeboxSystem : EntitySystem
+{
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly UserInterfaceSystem _ui = default!;
+    [Dependency] private readonly DeviceLinkSystem _link = default!;
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+
+    public const string PortSongPlayed = "Start";
```

Comment:
These really shouldn't be hardcoded I don't think, but making them variables makes checking them a pain...